bedrock/products/templates/products/vpn/more/what-is-a-vpn-v2.html

```diff
@@ -0,0 +1,58 @@
+
+<h3 class="c-section-heading">What is Mozilla VPN?</h3>
+<p>A VPN, or Virtual Private Network, is a tool that allows you to access the internet securely wherever you are. Tools
+like Mozilla VPN are available to anyone who wants to protect their connection to the internet, especially when it’s
+over insecure networks like public WiFi. If you don’t want people being able to see where you’re located when you’re
+online, you need a VPN. It’s even useful on a home network—without a VPN, websites and your Internet Service Provider
+can see your Internet traffic, and even the sites you’re browsing.</p>
+
+<h3 class="c-section-heading">How does Mozilla VPN work?</h3>
+<p>Mozilla VPN helps protect your data online by creating a secure “tunnel” between your device and the VPN or site you’re
+visiting. This “tunnel” protects you in two key ways:</p>
+<ul class="mzp-u-list-styled">
+  <li>When enabled, this tunnel will encrypt the traffic—in other words, encode your activity on the Internet—between you and
+  Mozilla VPN so that no one on your local WiFi can see it.</li>
+  <li>It’ll handle traffic from all apps and software running on your device, so you can browse and communicate freely over a
+  secure private network connection. <strong>Without a VPN, your network traffic could be monitored by someone on the same network
+  as your device.</strong></li>
+</ul>
+<p>You can also avoid trackers with VPN. How does this help you? Well, it can help cut down on intrusive advertising—you
+know, where you click a link once and all of a sudden it feels like you’re getting bombarded by ads that know everything
+about you. <strong>VPN can help you cut down on that terrible advertising.</strong></p>
+<p>Mozilla VPN will also hide your IP, or Internet Protocol, address, which can help hide your location and protect your
+identity. A VPN provider typically offers a number of connection gateways in different locations, which allows you to
+use an IP address from a location different from your own.</p>
+<p>That way, whatever site or service you’re connecting to sees the VPN’s IP, not your actual IP address, as the source of
+your traffic. Mozilla VPN, for example, routes your traffic through a secure server and lets you <strong>choose from more than
+30 countries through which you can route your traffic.</strong></p>
+<p>In short, Mozilla VPN is a tool that allows you to take back control of your online activity from corporations, hackers,
+and thieves who would love to get a chance to steal your data or your identity, or otherwise compromise your online
+life. By subscribing to Mozilla VPN, you’re also supporting our mission as a company to make the Internet a better place
+for all:</p>
+<p><strong><a href="{{ url('mozorg.about.manifesto') }}#principles">Principle 4:</a> Individuals’ security and privacy on the internet are fundamental and must not be treated as optional.</strong></p>
```

Comment:
Since #10426 merged we can link directly to Principle 4:
```suggestion
<p><strong><a href="{{ url('mozorg.about.manifesto') }}#principles-04">Principle 4:</a> Individuals’ security and privacy on the internet are fundamental and must not be treated as optional.</strong></p>
```